core-s3/src/test/scala/io/aiven/guardian/kafka/s3/Main.scala

```diff
@@ -80,12 +80,8 @@ object Entry extends LazyLogging {
       system: ActorSystem,
       s3Settings: S3Settings
   ): Future[Set[String]] =
-    // Bug that needs to be fixed upstream in Alpakka, this specific S3 api call is not region specific
```

Comment:
This has already been fixed in Pekko, see https://github.com/apache/incubator-pekko-connectors/pull/66